modules/dnn/src/op_inf_engine.hpp

```diff
@@ -28,6 +28,7 @@
 #define INF_ENGINE_RELEASE_2021_4 2021040000
 #define INF_ENGINE_RELEASE_2022_1 2022010000
 #define INF_ENGINE_RELEASE_2023_0 2023000000
+#define INF_ENGINE_RELEASE_2024_0 2024000000
```

Comment:
```suggestion
#define OPENVINO_RELEASE_2024_0 2024000000
```
and can we remove old ones?

Reply:
Let’s remove old ones, but keep IE notation to reduce changes.

Reply:
done